docs/getting-started/linux-installer.md

```diff
@@ -230,11 +246,81 @@ applicable for `td-agent` versions 4.1 or newer):
   sudo systemctl restart td-agent
   ```
 
+### Auto Instrumentation
+
+#### Installation
+
+To install the Collector and the Splunk OpenTelemetry Auto Instrumentation for
+Java packages, run the installer script with the `--with-instrumentation`
+option:
+```sh
+curl -sSL https://dl.signalfx.com/splunk-otel-collector.sh > /tmp/splunk-otel-collector.sh && \
+sudo sh /tmp/splunk-otel-collector.sh --with-instrumentation --realm SPLUNK_REALM -- SPLUNK_ACCESS_TOKEN
+```
+
+To automatically define the optional `deployment.environment` resource
+attribute at installation time, run the installer script with the
+`--deployment-environment VALUE` option (replace `VALUE` with the desired
+attribute value, e.g. `prod`):
+```sh
+curl -sSL https://dl.signalfx.com/splunk-otel-collector.sh > /tmp/splunk-otel-collector.sh && \
+sudo sh /tmp/splunk-otel-collector.sh --with-instrumentation --deployment-environment VALUE --realm SPLUNK_REALM -- SPLUNK_ACCESS_TOKEN
+```
+
+**Note:** After successful installation, the Java application(s) on the host need to be
+manually started/restarted.
+
+#### Post-Install Configuration
+
+- The `/etc/ld.so.preload` file will be automatically updated with the path to
+  the installed instrumentation library, i.e.
```

Comment:
let's use `e.g.` (for example) instead of `i.e.` (in other words) here

Reply:
> let's use `e.g.` (for example) instead of `i.e.` (in other words) here

Well, the default installation path of the .so library file shouldn't change.  I can just change it for that instead.